src/Features/CSharp/CSharpFeaturesResources.resx

```diff
@@ -228,6 +228,9 @@
   <data name="Deprecated" xml:space="preserve">
     <value>deprecated</value>
```

Comment:
Yes, this is actually a regression that we haven't implemented yet. FWIW, I am really not sure about this Obsolete text change for a few reasons:

1. It's pretty gratuitous. I'm not sure I understand the expected user value.
2. We still need to show "deprecated" for methods using the DeprecatedAttribute. With this change, Obsolete and Deprecated will be unnecessarily different, and I don't think we want that.
3. It seems like the whole point of this change is to make "[deprecated]" look like an attribute (i.e. [Obsolete]), but that was never the design. If we do this, it may appear to be a bug that we don't show all of the other attributes for a member. However, adding all attributes is a [very bad idea](https://github.com/dotnet/roslyn/blob/master/src/VisualStudio/CSharp/Impl/CSharpPackage.cs#L30-L47).

I'm inclined not to take this change.